api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java

```diff
@@ -125,6 +125,247 @@ public static String inTBProgramWithinReportingPeriodAtLocation(Integer tbProgra
         tbProgramId);
   }
 
+  /**
+   * Patients with Pulmonary TB Date in Patient Clinical Record of ART date TB
+   *
+   * @param encounterTypeId
+   * @param pulmonaryTBConcept
+   * @param yesConcept
+   * @return
+   */
+  public static String pulmonaryTB(
+      Integer encounterTypeId, Integer pulmonaryTBConcept, Integer yesConcept) {
+    return String.format(
+        "SELECT p.patient_id FROM patient p INNER JOIN encounter e"
+            + "ON p.patient_id = e.patient_id"
```

Comment:
Queries need blank space whenever breaking on new lines since + is concatenation.